cypress/cypress/e2e/bceid.feature

```diff
@@ -0,0 +1,67 @@
+Feature: BceID User Tests
+
+  Deals with BCeID Business user scenarios
+
+  @loginAsBCeID
+  Scenario: BceID User Login
+    Given I am a "BceID" user    
+    When I can read "New client application"
+    
+  @loginAsBCeID
+  Scenario: BceID Unregistered User
+    Given I am a "BceID" user
+    When I can read "New client application"
+    And I select "DMH - 100 Mile House Natural Resource District" from the "District" form input
+    Then I mark "I have an unregistered sole proprietorship" on the "Type of business" "radio" input
+    And I fill the form as follows
+      | Field name | Value                     | Type   |      
+      | Year       | 1990                      | text   |
+      | Month      | 01                        | text   |
+      | Day        | 01                        | text   |
+    Then I click on next
```

Comment:
That's irrelevant as all the prefixes are just **Step** and Cypress will handle any prefix the same way. The **Then** here could be anything

Reply:
I know they work just the same way.
But for readability and for the expect meaning of what each sentence is trying to achieve, I think it matters and it can help.
https://cucumber.io/docs/gherkin/reference/#given
Even your template gives a good, quick explanation about it.
            Given [some initial context]
            When [some event occurs]
            Then [some outcome should occur]

If needed I can help update that.

Reply:
You don't need to change anything as this file is not an example file but, instead, a file I created to use as a final test file. As long as it runs as expected, it should be fine.

Feel free to change it if you want but it is not required as it is not a demo or example file. The example file is called `sample.feature` as described on the readme file.